src/button.ts

```diff
@@ -197,13 +197,17 @@ export default class Button extends LitElement {
     }
   }
 
+  /* v8 ignore start */
   #onPrefixIconSlotChange() {
     const assignedNodes = this.#prefixIconSlotElementRef.value?.assignedNodes();
     this.hasPrefixIcon = Boolean(assignedNodes && assignedNodes.length > 0);
   }
+  /* v8 ignore end */
```

Comment:
An alternative to tests like [this](https://github.com/CrowdStrike/glide-core/blob/main/src/button.test.basics.ts#L27-L41).